docs/csharp/whats-new/tutorials/snippets/staticinterfaces/GetDuplicated.cs

```diff
@@ -0,0 +1,4 @@
+﻿public interface IGetDuplicated<T> where T : IGetDuplicated<T>
+{
+    static virtual T operator ++(T other) => throw new NotImplementedException();
```

Comment:
I'd like to think through this and use a different example for a `static virtual` (not `abstract`) method. That's because in practice, if the only reasonable implementation in the interface declaration is to `throw` a `NotImplementedException`, that indicates that it should be `abstract`.

Maybe consider an interface that provides `<` and `>`, but assumes that `T` implements `IComparable<T>`. Something like:

```csharp
public interface IArithmeticComparable<T> where T : IComparable<T>
{
    static virtual bool operator > (T left, T right) => left.CompareTo(right) > 0;

// etc.
}
```

I know that adds a bit more concepts to explain, so I'm open to any alternative example. That's just the first obvious one that came to my mind.

Reply:
That is valid point, @BillWagner, thanks! But it looks like my knowledge is missing something, because my question is: What would be the use case of such implementation? In other words how to call the static virtual defined in an interface without any definition in the type implementing that interface? I know I'm asking kinda for an implementation details, but can you point me some examples of such use cases or implementations?